panel/src/components/Forms/Field/PasswordField.vue

```diff
@@ -5,7 +5,9 @@
     v-bind="$props"
     class="k-password-field"
   >
-    <slot slot="options" name="options" />
+    <template #options>
+      <slot name="options" />
+    </template>
```

Comment:
Is this correct? I haven't looked this up in the Vue docs, but I assume the `slot` attribute on the `<slot>` element just didn't do anything until now. If that's the case, we should just remove that attribute.

Reply:
Let's check indeed, but I assumed the old syntax was doing what it was supposed to do: just pass on its own slot to the child component's slot with the same name.